civis/civis-aqueduct-utils/civis_aqueduct_utils/github.py

```diff
@@ -0,0 +1,36 @@
+import base64
+import fsspec
+import requests
+
+# Function to overwrite file in GitHub
+def upload_file_to_github(token, repo, branch, path, local_file_path, commit_message):
+    BASE = "https://api.github.com"
+
+    # Get the sha of the previous version
+    r = requests.get(
+        f"{BASE}/repos/{repo}/contents/{path}",
+        params={"ref": branch},
+        headers={"Authorization": f"token {token}"},
+    )
+    r.raise_for_status()
+    sha = r.json()["sha"]
+
+    # Upload the new version
+    with fsspec.open(local_file_path, "rb") as f:
+        contents = f.read()
+
+    r = requests.put(
+        f"{BASE}/repos/{repo}/contents/{path}",
+        headers={"Authorization": f"token {token}"},
+        json={
+            "message": commit_message,
+            "committer": {

```

Comment:
Just had a thought: we should probably make this "commiter" field a parameter as well, with this dictionary as the default value. So we could have at the top of this file
```
DEFAULT_COMMITTER = {
  "name": "Los Angeles ITA data team",
  "email": "ITAData@lacity.org",
}
```
and then in the function parameter list have `commiter=DEFAULT_COMMITER`